content/management/configuration-api/index.mdx

```diff
@@ -1063,11 +1065,12 @@ Creates a new bot. To make the bot available for receiving chats or webhooks, us
 
 __*__ Possible values for the `groups[].priority` parameter:
 
-| Possible value |                                                                                                                                        Notes                                                                                                                                        |
-| -------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
-| `first`        |                           The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents. Corresponds to the **Bot** priority option in the LiveChat App.                           |
-| `normal`       | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. Corresponds to the **Primary** priority option in the LiveChat App. |
-| `last`         |              The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group. Corresponds to the **Secondary** priority option in the LiveChat App.               |
+| Possible values |                                                                                                      Notes                                                                                                      |
+| --------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| `first`         |                         The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                         |
+| `normal`        | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. |
+| `last`          |               The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group.                |
+| `supervisor`    |                                                                       Bot works as `supervisor` so it will not be assigned to any chats.                                                                        |
```

Comment:
As I see we were consistent with the possible values being the same as for Agents for the Bot-related values as well 🤔  I _think_ we should stick to that and keep:
```suggestion
| `supervisor`    |                                                                       Agents with the `supervisor` priority will not get any chats assigned automatically.                                                                        |
```
but I will need to consult this with Ola and let you know :)